src/app/content/studyGuides/components/UsingThisGuide/UsingThisGuideBanner.tsx

```diff
@@ -82,14 +88,32 @@ const BodyWrapper = styled.div`
   display: flex;
   justify-content: center;
 `;
-
 interface Props {
   onClick: () => void;
-  isUTGopen: boolean;
+  show: boolean;
+  isOpenedForTheFirstTime: boolean;
 }
 
 // tslint:disable-next-line:variable-name
 const UsingThisGuideBanner = (props: Props) => {
+  const desktopBannerRef = React.useRef<HTMLImageElement>(null);
+  const toggleCounter = React.useRef(0);
+  const trackOpenUTG = useAnalyticsEvent('openUTG');
+
+  React.useEffect(() => {
+    if (props.show) {
+      toggleCounter.current += 1;
+      Cookies.set(cookieUTG, 'true');
+    }
+    // Send GA event when banner is open, except when it is opened initially.
+    if (props.show && (!props.isOpenedForTheFirstTime || toggleCounter.current > 1)) {
+      trackOpenUTG();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [props.show]);
+
+  if (!props.show) { return null; }
```

Comment:
Could it be refactored a bit? The ref counter feels unnecessary.

I think it could be achieved by:
- Render Banner conditionally in its parent
- In an `useEffect` with empty dependency array check for the cookie - if it's not there, focus the banner, if its there then `trackOpenUTG()`

Reply:
Focus is no longer being implemented right now, re factored, I think it works as expected.